src/rebar_git_resource.erl

```diff
@@ -68,28 +68,28 @@ parse_git_url("https://" ++ HostPath) ->
     [Host | Path] = string:tokens(HostPath, "/"),
     {Host, filename:rootname(filename:join(Path), ".git")}.
 
-download(Dir, {git, Url}) ->
+download(Dir, {git, Url}, State) ->
```

Comment:
It isn't used, but required to be passed around because the `download/3` callback requires it. It isn't `_State` here simply because this function calls `download/3` again. We could have a `rebar_git_resource:download/3` that calls `download/2` I guess.